applications/openshift/api-server/api_server_audit_log_path/oval/shared.xml

```diff
@@ -20,7 +20,7 @@
 
   <ind:textfilecontent54_object id="object_api_server_audit_log_path" version="1">
     <ind:filepath>/etc/origin/master/master-config.yaml</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*auditConfig:\:(?:[^\n]*\n+)+?[\s]*auditFilePath\:[\s]+(\S+)[\s]*$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*auditConfig\:(?:[^\n]*\n+)+?[\s]*auditFilePath\:[\s"']+(\S+)[\s"']*$</ind:pattern>
```

Comment:
While an improvement, this regex will match the final ``"``.

For example, if the config stanza is:
`````
auditConfig:
  auditFilePath: "/var/log/audit/ocp.log"
`````

Then the OVAL will match on ``/var/log/audit/ocp.log"``

Played around with the capture group but couldn't figure out how to only parse text within the quotes (if quotes are present).

Reply:
We can use a negation in a bracketed character class for the capture group to capture anything that isn't a space or quote character like this snippet:

  auditFilePath\:[\s"']+([^\s"']+)[\s"']*$

I will revise the patch to use this approach.